### DISCUSSION TITLE
NVIDIA Container High Power usage CPU

### DISCUSSION THREAD

**Discussion by francesco1001** [LizardByte/community#194] [Sunshine] [answered]
I have noticed since switching to sunshine from gamestream the following issues. I am unsure if this is normal:
1. NVIDIA container starts with PC when sunshine service is enabled on startup. Uses 10% of CPU constantly, does not stop. increasing idle temps by 20 degrees C
2. If sunshine serivce is not started automatically and instead done manually, NVIDIA container does not turn on until moonlight connects to the server, however this process continues after disconnection using resources.
3. I have tried uninstalling NIVIDA gamestream, disabling overlay features
4. Manually ending the task does not seem to affect moonlight play but is annoying to keep doing so I have just disabled sunshine startup and manually stopping the task. So I assume its not a required service.

PC Specs:
Processor	Intel(R) Core(TM) i9-10900KF CPU @ 3.70GHz   3.70 GHz
Installed RAM	64.0 GB (63.8 GB usable)
Nvidia 3090 video card
Windows 11 latest

**Comment by Cr0w1ey** [comment, marked as answer]
Known issue with latest nVidia driver

https://twitter.com/tomwarren/status/1632680688968138752
